### ISSUE TITLE
accessibility changes

### ISSUE THREAD

**Issue by kencheeto** [zendesk/linksf#213]
From feedback:

(1) None of the "links" to services on the home page are links. They are a collection  of elements that can't receive keyboard focus, and therefore a user who can't use a pointer device can't get beyond the first page.

(2) Many of the images on search results are <i> tags, not actual images. As such, they have no alt attribute describing what they are -- so assistive technologies don't know what they are. Voiceover just reads them as "link" "link" "link" -- not "Housing" "Food" "Medical".
